graylog2-web-interface/src/components/event-definitions/replay-search/LinkToReplaySearch.tsx

```diff
@@ -21,12 +21,12 @@ import Routes from 'routing/Routes';
 import { ReplaySearchButtonComponent } from 'views/components/widgets/ReplaySearchButton';
 import useParams from 'routing/useParams';
 
-const LinkToReplaySearch = ({ isEvent = false, id }: { id?: string, isEvent?: boolean }) => {
+const LinkToReplaySearch = ({ isEvent = false, id, onClick }: { id?: string, isEvent?: boolean, onClick?: () => void }) => {
   const { definitionId } = useParams<{ alertId?: string, definitionId?: string }>();
```

Comment:
Had a chat about these linter hints with Dennis and started using `undefined` as a default value. In every case we want to enable this rule, but there is no elegant solution for handling the undefined case.